File: go/1013.go
// Funcional no beecrowd
package main

import (
	"fmt"
	"math"
)

func main() {
	var a float64
	var b float64
	var c float64

	fmt.Scan(&a)
	fmt.Scan(&b)
	fmt.Scan(&c)

	ab := (a + b + math.Abs(a-b)) / 2
	abc := (ab + c + math.Abs(ab-c)) / 2
	fmt.Println(abc, "eh o maior")
}

File: Lua/Iniciante/1013.lua
-- Funcional no beecrowd
local A = io.read('n')
local B = io.read('n')
local C = io.read('n')

local D = (A + B + math.abs(A - B)) / 2
local E = (D + C + math.abs(D - C)) / 2

print(string.format("%d eh o maior", E))